rasa/nlu/test.py

```diff
@@ -1071,15 +1214,21 @@ def align_entity_predictions(
     for p in result.entity_predictions:
         entities_by_extractors[p[EXTRACTOR]].append(p)
     extractor_labels: Dict[Text, List] = {extractor: [] for extractor in extractors}
+    extractor_confidences: Dict[Text, List] = {
+        extractor: [] for extractor in extractors
+    }
     for t in result.tokens:
         true_token_labels.append(_concat_entity_labels(t, result.entity_targets))
         for extractor, entities in entities_by_extractors.items():
-            extracted = _concat_entity_labels(t, entities, {extractor})
-            extractor_labels[extractor].append(extracted)
+            extracted_labels = _concat_entity_labels(t, entities, {extractor})
+            extracted_confidences = _get_entity_confidences(t, entities, {extractor})
+            extractor_labels[extractor].append(extracted_labels)
+            extractor_confidences[extractor].append(extracted_confidences)
 
     return {
         "target_labels": true_token_labels,
         "extractor_labels": dict(extractor_labels),
+        "confidences": dict(extractor_confidences),
```

Comment:
Since `extractor_labels` and `extractor_confidences` are `dict`s already, could we remove the calls to `dict`? Or is there a reason we're doing this shallow copy?